specs/phase0/p2p-interface.md

```diff
@@ -691,9 +691,9 @@ The fields are, as seen by the client at the time of sending the message:
     - `current_fork_version` is the fork version at the node's current epoch defined by the wall-clock time
       (not necessarily the epoch to which the node is sync)
     - `genesis_validators_root` is the static `Root` found in `state.genesis_validators_root`
-- `finalized_root`: `state.finalized_checkpoint.root` for the state corresponding to the head block
+- `finalized_root`: `store.finalized_checkpoint.root` according to [fork choice](./fork-choice.md).
```

Comment:
This is a no-op, right? The `state` here is implicitly your head which is going to share the store's finalized checkpoint by definition

Reply:
no, there may be a reorg that unrealizes finality in `state` that was already realized on a different branch - `store.finalized_checkpoint` tracks the highest finality across all branches, while `state` just tracks the one from the branch currently selected by fork choice (which may be lower).